inst/generate.test.data.R

```diff
@@ -109,12 +109,14 @@ simple.pgs.application.test.data <- list(
     pgs.weight.data = data.frame(
```

Comment:
Consider moving this into a file under `tests/testthat` with a `helper-` prefix. Something like `helper-generate.test.data.R`. These test helpers are loaded into the namespace only when tests run, and keeps them separate from package functions. You don't need to call this directly in your tests, but it still may be a better place to keep the function for your use in future.